src/main/java/com/google/api/codegen/config/DiscoGapicInterfaceContext.java

```diff
@@ -302,4 +323,27 @@ public String serviceTitle() {
   public String getServiceAddress() {
     return getDocument().baseUrl();
   }
+
+  // TODO(andrealin): Parameterize this in Config instead of hard-coding it.
+  @Override
+  public String getOperationServiceName() {
+    return getOperationsScopeName() + "Operation";
+  }
+
+  private String getOperationsScopeName() {
+    // Hard-code Compute API's GlobalOperationClient in for now.
+    // We can expose this configuration in the GAPIC config later.
+    if (getLongRunningMethods().isEmpty()) {
+      return "$ Failed to find methods to be used by long running client.";
```

Comment:
Hmm, I don't want to throw an exception if someone accidentally calls this method and intends it to be a no-op.

Reply:
Why would somebody call a method to do no-op? Methods should not be called, if they are not needed. Throwing exception allows to fail fast. In general throwing exception is not bad (any error, thrown explicitly can be immediately fixed/handled, this is better than silently hiding it, so it eventually pops up somewhere else in the most unexpected moment).

Reply:
Oh, I had been looking at other examples in the codebase where methods are unimplemented so we return a `"$ NOT IMPLEMENTED` string instead of throwing an error.

Here I suppose we don't have to do that if every time we call this method, we check beforehand that it is an LRO method.

Done.